great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -1595,6 +1595,61 @@ def _determine_default_action_list(self) -> Sequence[ActionDict]:
 
         return Checkpoint.DEFAULT_ACTION_LIST
 
+    @public_api
+    @new_argument(
+        argument_name="id",
+        version="0.15.48",
+        message="To be used in place of `ge_cloud_id`",
+    )
+    def get_checkpoint(
```

Comment:
I might change this to `get_legacy_checkpoint` (same with delete) to indicate it is going away.